make formats docs folder, create COCO format documentation (#241)
openvinotoolkit/datumaro: docs/formats/mnist_user_manual.md

              --output-format imagenet \
              --filter '/item[annotation/label="3"]'
```

> Some examples of working with Pascal VOC dataset from code you can found in

Reviewer: ```suggestion
More examples of working with MNIST dataset from code can be found in
```
